builder/llpcBuilder.cpp

```diff
@@ -492,3 +492,44 @@ Type* Builder::GetBuiltInTy(
     }
 }
 
+// =====================================================================================================================
+// Get a constant of FP or vector of FP type from the given APFloat, converting APFloat semantics where necessary
+Constant* Builder::GetFPConstant(
```

Comment:
GetFPConstant -> GetFloatingPointConstant

Reply:
That would be unnecessarily verbose; everyone knows what "FP" means. There is an LLVM class ConstantFP, so the use of "FP" is consistent with that.

Reply:
Let's change it to GetFpConstant